lib/views/pages/dashboard.dart

```diff
@@ -50,29 +50,6 @@ class Dashboard extends HookConsumerWidget {
         alignment: Alignment.center,
         child: Column(
           children: [
-            TransactionSummary(
-              totalBalance: totalIncome - totalExpenses,
-              totalIncome: totalIncome,
-              totalExpenses: totalExpenses,
-              currency: currency,
-            ),
-            FilledButtonText(
-              text: 'Go To Notifications Page',
-              onPressed: () => {navigateTo(context, Routes.notifications)},
-            ),
-            OutlinedButtonText(
-              text: 'Outlined Pink',
-              color: AppColor.pink,
-              onPressed: () => {},
-            ),
-            OutlinedButtonText(
-              text: 'Outlined Blue',
-              onPressed: () => {},
-            ),
-            ElevatedButton(
-              onPressed: () => {navigateTo(context, '/account_settings')},
-              child: const Text('Go To Settings'),
-            ),
```

Comment:
@louisepujante pls confirm if na delete by accident or idelete na jud ni? kay ang summary mawala and i think dapat dili kay naay PR ani before but not sure huhu pa confirm lang kow 🙇

Reply:
@AubreyV aahhhh okay miss, will restore it 🙇‍♀️

Reply:
@AubreyV @louisepujante 
All reusable widgets should be in the Notifications page, so it's okay to remove this. I also commented in another PR to remove